src/Catel.MVVM/Catel.MVVM.Shared/Windows/Windows/DataWindow/DataWindow.cs

```diff
@@ -770,8 +771,10 @@ protected override void OnContentChanged(object oldContent, object newContent)
                 return;
             }
 
+            var wrapControlService = ServiceLocator.Default.ResolveType<IWrapControlService>();
```

Comment:
Store in a static field. If people want to replace, they'll have to do in their app.xaml.cs